react-components/src/common/i18n/en/reveal-react-components.json

```diff
@@ -39,10 +39,13 @@
   "POINT_CLOUDS": "Point clouds",
   "RESET_CAMERA_TO_HOME": "Reset camera to home",
   "RULESET_SELECT_HEADER": "Select color overlay",
+  "RULESET_NO_SELECTION": "No RuleSet Selected",
   "SCENE_SELECT_HEADER": "Select 3D location",
   "SEARCH_PLACEHOLDER": "Search",
   "SELECT_INDIVIDUAL_3D_MODELS": "Select individual models",
   "SETTINGS_TOOLTIP": "Settings",
+  "SHOW_LESS": "Show less",
+  "SHOW_MORE": "Show more",
```

Comment:
I guess these translations won't be available for June... But good to put them here anywhere

Reply:
yep